res/controllers/Numark_DJ2GO2_Touch_scripts.js

```diff
@@ -0,0 +1,210 @@
+    // using Function.prototype.call to assign the custom Deck being constructed
+    // to 'this' in the context of the generic components.Deck constructor
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/call
+    components.Deck.call(this, deckNumbers);
+
+    this.playButton = new components.PlayButton([0x90 + midiChannel, 0x00]);
+    this.cueButton = new components.CueButton([0x90 + midiChannel, 0x01]);
+    this.syncButton = new components.SyncButton([0x90 + midiChannel, 0x02]);
+
+    this.pflButton = new components.Button({
+        midi: [0x90 + midiChannel, 0x1B],
+        key: "pfl"
+    });
+
+    this.loadButton = new components.Button({
+        midi: [0x9F, 0x02 + midiChannel],
+        key: "LoadSelectedTrack",
+        input: function(channel, control, value, status, _group) {
+            this.send(this.isPress(channel, control, value, status) ? this.on : this.off);
+            components.Button.prototype.input.apply(this, arguments);
+        }
+    });
+
+    this.preGain = new components.Pot({
+        midi: [0xB0 + midiChannel, 0x16],
+        group: "[QuickEffectRack1_" + this.currentDeck + "]",
+        key: "super1"
+    });
+
+    engine.setValue(this.currentDeck, "rate_dir", -1);
+    this.tempoFader = new components.Pot({
+        group: "[Channel" + deckNumbers + "]",
+        midi: [0xB0 + midiChannel, 0x09],
+        key: "rate",
+        inSetParameter: components.Pot.prototype.inSetParameter,
+        connect: function() {
+            engine.softTakeover(this.group, "rate", true);
+            components.Pot.prototype.connect.apply(this, arguments);
+        }
+    });
+
+    this.hotcueButtons = [];
+    this.samplerButtons = [];
+    this.beatloopButtons = [];
+    for (var i = 1; i <= 4; i++) {
+        this.hotcueButtons[i] = new components.HotcueButton({
+            group: "[Channel" + deckNumbers + "]",
+            midi: [0x94 + midiChannel, 0x01 + i],
+            number: i,
+        });
+        this.samplerButtons[i] = new components.SamplerButton({
+            group: "[Sampler" + (i + (midiChannel * 4)) + "]",
+            midi: [0x94 + midiChannel, 0x31 + i],
+            number: i + (midiChannel * 4),
+        });
+        this.beatloopButtons[i] = new components.Button({
+            group: "[Channel" + deckNumbers + "]",
+            midi: [0x94 + midiChannel, 0x11 + i],
+            number: i,
+            key: "beatloop_" + Math.pow(2, i-1) + "_toggle"
+        });
+    }
+
+
+    this.loopIn = new components.Button({
+        midi: [0x94 + midiChannel, 0x21],
+        key: "loop_in",
+    });
+
+    this.loopOut = new components.Button({
+        midi: [0x94 + midiChannel, 0x22],
+        key: "loop_out",
+    });
+
+    this.LoopToggleButton = new components.LoopToggleButton([0x94 + midiChannel, 0x23]);
+
+    this.wheelTouch = function(channel, control, value, _status, _group) {
+        if ((_status & 0xF0) === 0x90) {
```

Comment:
```suggestion
    this.wheelTouch = function(channel, control, value, status, _group) {
        if ((status & 0xF0) === 0x90) {
```
only prefix parameters you don't use with an underscore